### PR TITLE
Completes PR #10686

### PR DESCRIPTION
**Pull request by yashkgp**
#### Brief description of what is fixed or changed
Changes Gas Constant as Avogadros number times Boltmannz's constant

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by smichr** [comment]
> Fixes Issue #13528

If the issue should be closed if this PR is merged then the word "Issue" should not appear between Fixes and the issue number.

File: sympy/physics/units/definitions.py [hunk 1]
```diff
@@ -153,7 +153,7 @@
 # vacuum impedance:
 Z0 = vacuum_impedance = Quantity("vacuum_impedance", impedance, u0 * c)
 # Coulomb's constant:
-coulomb_constant = coulombs_constant = electric_force_constant = Quantity("coulomb_constant", force*length**2/charge**2, 1/(4*pi*vacuum_permittivity), abbrev="k_e")
+Ke=coulomb_constant = coulombs_constant = electric_force_constant = Quantity("coulomb_constant", force*length**2/charge**2, 1/(4*pi*vacuum_permittivity), abbrev="k_e")
```
Comment (smichr) [inline sympy/physics/units/definitions.py]: space around "="

Comment (smichr) [inline sympy/physics/units/definitions.py]: The symbol is shown with lower case "k" [here](https://en.wikipedia.org/wiki/Coulomb%27s_constant).

Comment (gxyd) [inline sympy/physics/units/definitions.py]: Space is still missing here.

I'm not a fan of seeing such huge line which make it really hard to see in editor. Can you please make lines <= 79 characters? Just for the `Ke` and `R`, no need to touch other units for now.

**Comment by gxyd** [comment]
Also the 'Fixes #NNNN' doesn't work. Please make sure to use 'Fixes #NNNN' or 'Resolves #NNNN' in your PR description.

**Comment by oscargus** [comment]
Should be straightforward to fix conflicts and add the new information on the form used in the rest of the file now (derivation of the gas constant from avogadro_number*boltzmann). Seems like most of the previous information is already there now.

**Comment by czgdp1807** [comment]
Considering [this comment](https://github.com/sympy/sympy/issues/13528#issuecomment-360829433), I am closing this PR. Thanks.

**Pull request closed without merging**